### PR TITLE
Update PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by abelsiqueira**
**Description**

Update the review steps of the PR template.

**Related issues**:
- none

**Instructions to review the pull request**

Try to check the installation instructions for any pull request.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -9,16 +9,15 @@
 
 <!--
 - Make a GitHub Access token here https://github.com/settings/tokens
-- `cp secrets.txt.example secrets.txt`
-- Update the value of GITHUB_TOKEN in secrets.txt
+- export GITHUB_TOKEN as an environment variable with the token above
```
Comment (jspaaks) [inline .github/PULL_REQUEST_TEMPLATE.md:12]: maybe add something about storing the token, since you can only read it once on github

Comment (abelsiqueira) [reply .github/PULL_REQUEST_TEMPLATE.md:13]: I have updated with more information, let me know what you think.

**Review by jspaaks — APPROVED** [review]
Thank you for the PR

**Comment by abelsiqueira** [comment]
Thanks for the review

**Pull request merged**